crates/cargo-gpu/src/linkage.rs

```diff
@@ -0,0 +1,30 @@
+//! Mainly for the Linkage struct, which is written to a json file. Previously in `spirv-tools-cli` but got moved here.
```

Comment:
As mentioned previously, I wonder if the reference to `spirv-builder-cli` will ultimately mostly be out of context?